compositor_pipeline/src/pipeline/decoder/video/ffmpeg_h264.rs

```diff
@@ -20,11 +20,11 @@ use tracing::{debug, error, span, trace, warn, Level};
 pub struct H264FfmpegDecoder;
 
 impl H264FfmpegDecoder {
-    pub fn new(
+    pub fn spawn(
```

Comment:
`new` is used in Rust for "factory functions", that return an instance of an object of that type. This doesn't do that. I agree with renaming it to `start_ffmpeg_decoder_thread`. What do you think about removing this struct? It can be just a function since this doesn't have to be constructed (drop impl is not needed for this struct).